oracle/assets/service_checks.json

```
@@ -11,6 +11,20 @@
             "server"
         ],
         "name": "Can Connect",
-        "description": "Returns OK if the integration can connect to the oracle database and can run all the queries, CRITICAL otherwise"
+        "description": "Returns OK if the integration can connect to the oracle database, CRITICAL otherwise"
+    },
+    {
+        "agent_version": "6.0.0",
+        "integration": "Oracle Database",
+        "check": "oracle.can_query",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "groups": [
+            "server"
```

& fix the existing one

```suggestion
            "host",
            "server"
```